packages/@sanity/desk-tool/src/panes/documentPane/documentPanel/header/header.tsx

```diff
@@ -187,38 +175,15 @@ export function DocumentPanelHeader(props: DocumentPanelHeaderProps) {
   const versionMenu = useMemo(
     () =>
       showVersionMenu && (
-        <div className={styles.versionSelectContainer} ref={versionSelectRef}>
-          <Button
-            fontSize={1}
-            iconRight={SelectIcon}
-            mode="bleed"
-            onMouseUp={ignoreClickOutside}
-            onClick={onTimelineOpen}
-            padding={2}
-            selected={isTimelineOpen && timelineMode === 'rev'}
-            text={
-              // eslint-disable-next-line no-nested-ternary
-              menuOpen ? (
-                <>Select version</>
-              ) : rev ? (
-                <TimelineButtonLabel rev={rev} />
-              ) : (
-                <>Current version</>
-              )
-            }
-          />
-        </div>
+        <Box marginX={1} style={{marginLeft: 'auto'}}>
```

Comment:
This could be moved to a `const` outside the component + a `@todo` to refactor it to a styled-component later

Reply:
Because this is in the `documentPane` section I'll leave it as is since I think there's already work being done elsewhere!